### ISSUE TITLE
App installation path

### ISSUE THREAD

**Issue by nikobeer** [Edirom/sourceImageCartographer#8]
App installs itself into subcollection /apps/edirom/. Default should be /apps/

**Comment by bwbohl** [comment]
I don't consider this a bug, I think all other edirom apps should be installed in apps/edirom, too ;-)

**Comment by nikobeer** [comment]
Yes, i know… but that maybe needs some further discussion with the other projects…
